Sources/Engine/Base/Parser.hpp

```diff
@@ -0,0 +1,108 @@
+/* A Bison parser, made by GNU Bison 3.0.2.  */
```

Comment:
oops, `Engine/Base/Parser.hpp` wasn't supposed to be committed, will push a fixed (and rebased) version in minute